packages/server/database/types/Meeting.ts

```diff
@@ -1,7 +1,7 @@
 import generateUID from '../../generateUID'
 import GenericMeetingPhase from './GenericMeetingPhase'
 
-export type MeetingTypeEnum = 'poker' | 'retrospective' | 'action'
+export type MeetingTypeEnum = 'poker' | 'retrospective' | 'action' | 'teamPrompt'
```

Comment:
should we remove this type and use PG-generated enum to keep one source of truth?